### DISCUSSION TITLE
Please fix this, i don't like this scroll bar.

### DISCUSSION THREAD

**Discussion by lorarianz** [lllyasviel/Fooocus#2418] [General]
![Screenshot from 2024-03-03 10-06-59](https://github.com/lllyasviel/Fooocus/assets/159459697/17581cba-f356-4e3f-be4a-3624ceee33f7)

Fooocus version 2.2.0

**Comment by lgwjames** [comment]
ok here

![Screenshot 2024-03-03 at 07-43-40 Fooocus 2 2 0](https://github.com/lllyasviel/Fooocus/assets/35565479/b30f4b59-0b67-4191-8c1d-9b1d23e3df3b)

**Comment by Hypersonic-Popcorn** [comment]
You have the same thing. You can scroll the lora area a bit and you'll see the scroll bar they're talking about.
I think the outer container needs to be larger or the weight control should be shorter.

**Reply by mashb1t** [reply]
this is a Firefox issue as the spinner controls on number inputs are calculated outside the width of the element  Putting css `width: 100%` to the inputs works, but also destroys other views...
Your frontend skills and feedback are welcome!

**Reply by Hypersonic-Popcorn** [reply]
I am indeed using Firefox. Unfortunately, I have no frontend skills, I just know they're called controls :)

**Comment by mashb1t** [comment]
Fixed in https://github.com/lllyasviel/Fooocus/pull/2431
